app/views/govuk_publishing_components/components/docs/share_links.yml

```diff
@@ -256,10 +256,45 @@ examples:
           icon: 'youtube'
         }
       ]
+  with_flexbox:
+    data:
+      flexbox: true
+      links: [
+        {
+          href: '/twitter-share-link',
+          text: 'Twitter',
+          icon: 'twitter'
+        },
+        {
+          href: '/instagram-share-link',
+          text: 'Instagram',
+          icon: 'instagram'
+        },
+                {
+          href: '/flickr-share-link',
+          text: 'Flickr',
+          icon: 'flickr'
+        },
+        {
+          href: '/facebook-share-link',
+          text: 'Facebook',
+          icon: 'facebook'
+        },
+        {
+          href: '/youtube-share-link',
+          text: 'YouTube',
+          icon: 'youtube'
+        },
+        {
+          href: '/other-share-link',
+          text: 'Anything else that might be included that could have quite a long name',
+          icon: 'other'
+        },
+      ]
   with_square_icons:
```

Comment:
I know this isn't strictly part of this PR, but would it be possible to change this to something like `with_flexbox_and_square_icons`? Only because `with_square_icons` might suggest that square icons can be used independently of any other layout options but I think it currently only displays correctly when used in conjunction with `flexbox: true`.

Reply:
Thanks, done 👍